### PR TITLE
Adding mode probabilities to modeid class

### LINKED ISSUE #107: Parsing of modeId results from server

### PR DESCRIPTION
**Pull request by OskarLiew**
Fixing bug when using new version of vibium with probability table. If there is no probability table the results will be an empty dataframe

closes #107

### CODE REVIEW

File: mvg/features/modeid.py [hunk 1]
```diff
@@ -28,7 +28,9 @@ def __init__(self, results, t_zone=None, t_unit=None):
         else:
             dict_for_df = self.results().copy()
             dict_for_emerging = dict_for_df.pop("mode_info")
+            dict_for_mode_proba = dict_for_df.pop("", {})
```
Comment (tuix) [inline mvg/features/modeid.py:31]: Why empty key? Should be mode_probabilities I guedd.

**Review by tuix — CHANGES_REQUESTED** [review]

File: mvg/features/modeid.py (hunk 1 above)
Comment (OskarLiew) [reply mvg/features/modeid.py:31]: That is a fantastic point

**Review by SergioMDCB — APPROVED** [review]
This PR includes the mode probabilities for the parsing of the results

**Comment by tuix** [comment]
Let's bump the MVG version

**Review by vnadhan — APPROVED** [review]

**Review by SergioMDCB — APPROVED** [review]

**Review by tuix — APPROVED** [review]

**Pull request merged**